apps/coscrad-frontend/src/components/resources/videos/video-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,20 @@
+import { ICategorizableDetailQueryResult, IVideoViewModel } from '@coscrad/api-interfaces';
+import { SinglePropertyPresenter } from '../../../utils/generic-components';
+import { ResourceDetailFullViewPresenter } from '../../../utils/generic-components/presenters/detail-views';
+import { convertMillisecondsToSeconds } from '../utils/math';
+
+export const VideoDetailFullViewPresenter = ({
+    lengthMilliseconds,
+    text: plainText,
+    name,
+    id,
+}: ICategorizableDetailQueryResult<IVideoViewModel>): JSX.Element => (
+    <ResourceDetailFullViewPresenter name={name} id={id}>
+        <SinglePropertyPresenter
```

Comment:
It would be great to have a more sort of "visual" interface for the video (LOL)

Reply:
[Here](https://www.pivotaltracker.com/story/show/184530937) is the story.

Reply:
I have linked the story in a comment in the code.